openfang_config/upstream-overrides/crates/openfang-api/static/js/pages/agents.js

```
@@ -0,0 +1,744 @@
+    tplProviders: [],
+    tplLoading: false,
+    tplLoadError: '',
+    selectedCategory: 'All',
+    searchQuery: '',
+
+    builtinTemplates: [],
+    
+    // Load templates from API
+    async init() {
+      await this.loadTemplates();
+    },
```

Biome (2.4.10)

[error] This method is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)